stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -179,6 +180,13 @@ func (m *Manager) UpdateTenantConfig(ctx context.Context, tu *TenantUpdate) (*Te
 		args = append(args, *tu.Status)
 	}
 
+	if tu.DistributionAccount != nil {
+		fields = append(fields, "distribution_account = ?")
+		args = append(args, *tu.DistributionAccount)
+
+		log.Ctx(ctx).Warnf("distribution account updated to %s", *tu.DistributionAccount)
```

Comment:
💡  Considering we'll have multiple tenants, it's important to log the tenantID here. Can you update the lof by adding it?